aem-authoring-toolkit-plugin/src/test/com/exadel/aem/toolkit/test/widget/PasswordWidget.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exadel.aem.toolkit.test.widget;
+
+import com.exadel.aem.toolkit.api.annotations.main.Dialog;
+import com.exadel.aem.toolkit.api.annotations.main.DialogLayout;
+import com.exadel.aem.toolkit.api.annotations.widgets.DialogField;
+import com.exadel.aem.toolkit.api.annotations.widgets.Password;
+
+@Dialog(
+        name = "test-component",
+        title = "test-component-dialog",
+        layout = DialogLayout.FIXED_COLUMNS
+)
+@SuppressWarnings("unused")
+public class PasswordWidget {
+
+    @DialogField
+    @Password(
+            emptyText = "test-password",
+            autocomplete = "on",
+            autofocus = true,
+            retype = "test-password",
```

Comment:
Pls change this to a like of `retype-password`. It does not affect functionality but it spoils the semantics. This attribute is exactly the "name" attr of a field where retyped passw is set